lib/pure/asyncnet.nim

```diff
@@ -780,6 +780,123 @@ proc isClosed*(socket: AsyncSocket): bool =
   ## Determines whether the socket has been closed.
   return socket.closed
 
+proc sendTo*(socket: AsyncSocket, data: pointer, dataSize: int, address: string,
```

Comment:
Does this only work with UDP packets? What happens if this is a TCP socket?

Reply:
I am no expert on this, but looking at the low-level code (asyncdispatch) on Windows, WSASendTo is used, in which the documentation specifies that it is generally used for connectionless sockets. However, there is a note saying that: if used on a connected socket, the address will be ignored and WSASendTo will behave the same as WSASend.

I haven't tested it either. I will test the behavior on Windows. Do you have a solution in mind or something to add?

Reply:
I tested it now on Windows and the behavior was as expected. That is, when compiled without "-d: danger" it displays the error message caused by the assert. When compiled with "-d: danger", regardless of the address and port used in sendTO, the message was correctly forwarded by the socket to the destination used in connect for TCP protocol.

Reply:
I will do the same test on linux.

Reply:
I tested on linux and it happens the same as in Windows. That is, there is no problem using sendTo on TCP connections, however, it is not the most recommended.

Can we end this?